src/Stripe.net/Infrastructure/Public/IStripeClient.cs

```diff
@@ -47,43 +47,6 @@ public interface IStripeClient
             CancellationToken cancellationToken = default)
             where T : IStripeEntity;
 
-        /// <summary>Sends a request to Stripe's API as a synchronous operation.</summary>
-        /// <param name="method">The HTTP method.</param>
-        /// <param name="path">The path of the request.</param>
-        /// <param name="options">The parameters of the request.</param>
-        /// <param name="requestOptions">The special modifiers of the request.</param>
-        /// <param name="cancellationToken">The cancellation token to cancel operation.</param>
-        /// <returns>The response as a StripeResponse.</returns>
-        /// <exception cref="StripeException">Thrown if the request fails.</exception>
-        StripeResponse RawRequest(
```

Comment:
Why don't we want to add this to `IStripeClient`? I get an error in my IDE when I try to use this in my playground project:
```
'IStripeClient' does not contain a definition for 'RawRequestAsync' and no accessible extension method 'RawRequestAsync' accepting a first argument of type 'IStripeClient' could be found (are you missing a using directive or an assembly reference?)
```

Reply:
I'm a bit bit of an opponent to .NET interfaces and their overuse. Interfaces are impossible to evolve in a backwards compatible way and most of the time don't serve a good purpose (testing is a popularly cited one but virtual methods on deal with that almost as well).

I expect we'll be putting more properties on `StripeClient` (service navigator) and we won't be able to change the interface every time we add a property (because that will be a breaking change).

I'd like to keep IStripeClient similar to ResponseGetter in Java or RequestSender in Node where it has minimum of methods to be able to send a request. Service implementations will still pass IStripeClient around and use it for their implementation but `IStripeClient` won't have everything that `StripeClient` itself does.

Reply:
Got it, I trust you on this front. :) Thanks for the explanation!